drainer/translator/mysql.go

```diff
@@ -275,19 +273,7 @@ func (m *mysqlTranslator) genDeleteSQL(schema string, table *model.TableInfo, co
 }
 
 func (m *mysqlTranslator) GenDDLSQL(sql string, schema string, commitTS int64) (string, error) {
-	ddlParser := parser.New()
-	ddlParser.SetSQLMode(m.sqlMode)
-	stmt, err := ddlParser.ParseOneStmt(sql, "", "")
-	if err != nil {
-		return "", errors.Trace(err)
-	}
-
-	_, isCreateDatabase := stmt.(*ast.CreateDatabaseStmt)
-	if isCreateDatabase {
-		return fmt.Sprintf("%s;", sql), nil
-	}
-
-	return fmt.Sprintf("use `%s`; %s;", schema, sql), nil
+	return fmt.Sprintf("%s;", sql), nil
```

Comment:
just return sql, nil
if you want <sql>;  you can just use `sql + ";" here for more clear and performance

Reply:
revealed